### DISCUSSION TITLE
不同的子域名解析的问题

### DISCUSSION THREAD

**Discussion by Lovest20018** [kkkgo/PaoPaoDNS#67] [问答和帮助] [answered]
用群晖自带的反代服务器反代了几个不同的子域名。通过frp https形式穿透带外网。
假如ht tps://1.xx.com:999能访问的情况下，htt ps://2.xx.com:999就不能正常访问提示，提示群晖页面不存在。过一会可能htt ps://2.xx.com:999正常，htt ps://1.xx.com:999却不行了。
试了下外网的解析都是正常的。
试着把RULES_TTL设置为1，然后在force_ttl_rules 设置了一条cncheck.xx.com@223.5.5.5
错误提示是404 Not Found了。
这个问题应该怎么解决呢？
问到这里顺便多问下，
对于frp穿透的https域名，重定向能否带上端口呢？adgh里面是没办法这样设置的。
比如我要让htt ps://1.xx.com:999重定向到ht tp://192.168.1.2:5000 
h ttps://2.xx.com:999重定向到h ttp://192.168.1.2:9999

更新：
用手机流量测试了下。发现问题依旧。。那应该不是本地dns的问题？
可是我在公司远程访问一切都正常。。。这真是怪了。。。

**Comment by kkkgo** [comment, marked as answer]
域名DNS解析与端口无关，你应该只关注解析结果，也就是域名对应的A/AAAA记录，剩下的是你的frp配置相关问题。

**Reply by kkkgo** [reply]
使用addinfo功能可以查看更多解析结果信息。
